### ISSUE TITLE
Raspberry Pi will not boot when connected to GTR board, and power is shared between devices

### ISSUE THREAD

**Issue by jonmstraub** [bigtreetech/BIGTREETECH-GTR-V1.0#9]
The 5.0V and 3.3V power pins in the Raspberry Pi header should likely not be allowed into the 5V and 3.3V busses on the GTR board.

Each board will be overdrawn on current causing a low power condition for the Raspberry Pi. There should be some isolation here.

**Comment by bakaufman** [comment]
Did you flip the ribbon cable and try?

**Comment by clowrey** [comment]
Yes this is correct.. Only connect TX, RX, GND between them for a safe connection... design is crazy to have a 40 pin header when only 3 pins are needed.

**Comment by DJDOUBLEG** [comment]
Cut the 3.3V pin!
If i use RPi2 then this pin not problem.
Now is use RPi 3B+ then not boot up when 3.3V is connected.

**Comment by jonmstraub** [comment]
> Cut the 3.3V pin!
> If i use RPi2 then this pin not problem.
> Now is use RPi 3B+ then not boot up when 3.3V is connected.

Thank you for this info, though I understand there are workarounds. The point of opening the issue was to give feedback to the manufacturer to correct the problem in future releases.

**Comment by Geraldblin** [comment]
Hi can someone help me configure this board thx
